Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.hub.docker.com/library/golang:1.15.3 AS builder
+FROM library/golang:1.15.3 AS builder
```

Comment:
I think that's the only one that we use in Metal3-dev-env CI tests.

Reply:
Thanks. Only Dockerfiles are changed now.